.circleci/config.yml

```diff
@@ -234,5 +275,20 @@ workflows:
       - automatic-dependency-update 
 
   danger:
+    when:
+      not:
+        equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
+    jobs:
+      - revenuecat/danger
+
+  automatic-release:
+    when:
+      not:
+        equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
-      - revenuecat/danger
\ No newline at end of file
+      - automatic-release:
+          filters:
+            tags:
+              ignore: /.*/
+            branches:
+              only: main
```

Comment:
Hmm I wonder if we would like to do this only once a week or possibly a day... Doing it for every merge might be noisy? That said, this should be mostly just ios/android bumps which will happen weekly I think, so this might be ok.

Reply:
That made me think... merging this PR will also trigger an automatic release. Do we want that? I guess we can close the PR if we don't want to release at that moment. Maybe we can check the labels of the PRs to release and skip the release if it's only test, CI, and those that are not really public facing changes?

Reply:
Ahh good catch! And yeah, if we can filter out those, it should save time.